src/common.rs

```diff
@@ -20,6 +22,15 @@ pub(crate) mod tests {
     use super::*;
     use std::path::PathBuf;
 
+    #[allow(dead_code)]
+    pub fn blank_file_entry(total_lines: usize) -> FileEntry {
```

Comment:
I don't think that we need this function. See the comment above.

Reply:
Will remove!